packages/bits/spec/components/panel/panel.e2e.ts

```diff
@@ -3,150 +3,117 @@ import { ISize } from "selenium-webdriver";
-            it("Should display footer", async () => {
-                expect(await panel.isFooterDisplayed()).toEqual(true);
-            });
-
-            it("Should display header", async () => {
-                expect(await panel.isHeaderDisplayed()).toEqual(true);
-            });
-        });
+        const panelHidden = Atom.find(PanelAtom, "nui-demo-hidden-panel");
 
         it("should hide/unhide the left pane when hideLeftPane is toggled", async () => {
-            expect(await panel.isPaneDisplayed("left")).toEqual(true);
-            await panel.closeSidePane();
-            expect(await panel.isPaneDisplayed("left")).toEqual(false);
+            expect(await panelHidden.isPaneDisplayed("left")).toEqual(true);
+            await panelHidden.closeSidePane();
+            expect(await panelHidden.isPaneDisplayed("left")).toEqual(false);
         });
-
     });
 
     describe("Floating", () => {
-        const panel = Atom.find(PanelAtom, "nui-demo-floating-panel");
-
-        beforeEach(async () => {
-            await Helpers.prepareBrowser(`${pageToken}/floating`);
-        });
+        const panelFloating = Atom.find(PanelAtom, "nui-demo-floating-panel");
 
         it("should not change width of center pane while displaying floating panel", async () => {
-            const oldCenterPaneSize: ISize = await panel.getCenterPaneElementSize();
-            await panel.hoverOnSidePane();
-            const newCenterPaneSize: ISize = await panel.getCenterPaneElementSize();
+            const oldCenterPaneSize: ISize = await panelFloating.getCenterPaneElementSize();
+            await panelFloating.hoverOnSidePane();
+            const newCenterPaneSize: ISize = await panelFloating.getCenterPaneElementSize();
             expect(oldCenterPaneSize.width).toEqual(newCenterPaneSize.width);
         });
-
     });
 
     describe("Resize", () => {
-        const panel = Atom.find(PanelAtom, "nui-demo-resizable-panel");
+        const panelResize = Atom.find(PanelAtom, "nui-demo-resizable-panel");
         const gutter = Atom.find(ResizerAtom, "nui-demo-resizable-panel");
 
-        beforeEach(async () => {
-            await Helpers.prepareBrowser(`${pageToken}/resize`);
-            await Helpers.disableCSSAnimations(Animations.ALL);
-        });
-
-        it("should have horizontal resize direction", async () => {
-            expect(await gutter.getResizeDirection()).toEqual("horizontal", "should have correct resize direction");
-        });
-
         it("should make side panel bigger", async () => {
-            const oldCenterPaneSize = await panel.getCenterPaneElementSize();
-            await gutter.moveRight(300);
-            const newCenterPaneSize = await panel.getCenterPaneElementSize();
-            await expect(oldCenterPaneSize.width).toEqual(newCenterPaneSize.width + 300);
+            const originalSize = await browser.manage().window().getSize();
+            const movePanel = 300;
```

Comment:
maybe `gutterMoveDistance`? The reason I suggest this is that the name `movePanel` sounds like a function name instead of a constant.

Reply:
Agree.